modules/administration/pages/content-lifecycle.adoc

```diff
@@ -35,20 +39,23 @@ The software channels you selected should now be showing.
 To create a new filter, click btn:[Create new Filter].
 . Click btn:[Add Environment].
 . In the [guimenu]``Environment Lifecycle`` dialog, give the first environment a name and a description, and click btn:[Save].
-The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods (``.``), hyphens (``-``) and underscores (``_``).
+The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods, hyphens, and underscores.
 . Continue creating environments until you have all the  environments for your lifecycle completed.
 You can select the order of the environments in the lifecycle by selecting an environment in the [guimenu]``Insert before`` field when you create it.
 
 
 
 == Filter Types
 
-{productname} allows you to create various types of filters to control the content on project build.
-This is the list of supported filters:
+{productname} allows you to create various types of filters to control the content used for building the project.
+Filters allow you to select which packages will, or will not, be included in the build.
```

Comment:
Even if technically correct, I'd remove the ", or will not," phrase.  Probably easier to read:

> which packages will be in- or excluded in the build.